### ISSUE TITLE
Testing OpenVINO Music Separtion with Audacity 3.4.2

### ISSUE THREAD

**Issue by petersampsonaudacity** [intel/openvino-plugins-ai-audacity#3]
I have just tested this on a couple of my favourite test tracks:
a)track 1 of The Beatles Sgt Pepper and 
b) Frank Sinatra's Come fly with me.  

With both of them the vocal separation is extremely impressive, very impressive indeed.  I just used 2-Stem on Sgt Pepper - but on Sinatra I tried 4-stem - here I expected to see more in the drums stem (but I need to listen more carefully before passing judgement).

I note that it takes a lot of processing time the 2 min Sgt Pepper too 2m30secs and the 3 min Sinatra took 4 mins - and it certainly makes by processor work hard (I used the default CPU setting) - not many things get the fan running on my HP Envy but this tool certainly does 😉

**Comment by RyanMetcalfeInt8** [comment]
@petersampsonaudacity -- Thank you for sharing!

**Comment by Tserklaes** [comment]
Hello
The installation was straightforward.
But separation doesn't work.

Audacity error logs:
15:29:04: Operation 'OpenVINO Music Separation' took 0,025000 seconds. Poll was called 2 times and took 0,000001 seconds. Yield was called 0 times and took 0,000000 seconds.

**Comment by RyanMetcalfeInt8** [comment]
> Hello The installation was straightforward. But separation doesn't work.
> 
> Audacity error logs: 15:29:04: Operation 'OpenVINO Music Separation' took 0,025000 seconds. Poll was called 2 times and took 0,000001 seconds. Yield was called 0 times and took 0,000000 seconds.

Hi @Tserklaes, could you open a new issue with some more details that include more details of your system setup (CPU version, Windows version, etc.). As well as some more details about what you mean by "doesn't work" (screenshots help alot here).

Thanks,
Ryan

**Comment by Meowdoleon** [comment]
> @petersampsonaudacity -- Thank you for sharing!
> 
> > I note that it takes a lot of processing time the 2 min Sgt Pepper too 2m30secs and the 3 min Sinatra took 4 mins - and it certainly makes by processor work hard (I used the default CPU setting) - not many things get the fan running on my HP Envy but this tool certainly does 😉
> 
> Yes, this feature uses Demucs v4 AI model (developed by Meta), and is quite heavy. Depending on your system, and if 'GPU' shows up in the drop-down list of devices, GPU may produce results faster than CPU.

Yup, can confirm that a three minutes 45 seconds song took me about three minutes to separate using an RX 580 GPU, and it did wonder. You can faintly hear the vocal in the instrumental and vice versa, but I think that is expected, especially from a song using synthesized voice as vocals.
